#request

---
#response
geometry_msgs/Pose2D[] frontier_list
